backend/medtagger/api/tasks/service_rest.py

```diff
@@ -45,3 +45,18 @@ def post() -> Any:
         tags = [LabelTag(tag['key'], tag['name'], tag['tools']) for tag in payload['tags']]
 
         return business.create_task(key, name, image_path, categories_keys, tags), 201
+
+
+@tasks_ns.route('/<string:task_key>')
+class Task(Resource):
+    """Endpoint that manages single task."""
+
+    @staticmethod
+    @login_required
+    @tasks_ns.marshal_with(serializers.out__task)
+    @tasks_ns.doc(security='token')
+    @tasks_ns.doc(description='Get category for given key.')
+    @tasks_ns.doc(responses={201: 'Success', 404: 'Could not find task'})
```

Comment:
I think we should use HTTP code 200 here. We're not creating a new task, so there's no point in using 201 code.

Reply:
You are right :) Copy-paste method failed me once more!